cpp/distributed/dist-mnist.cpp

```diff
@@ -89,8 +82,10 @@ int main(int argc, char* argv[]) {
   // Number of epochs
   size_t num_epochs = 10;
 
+  /*
   // start timer
   tstart = MPI_Wtime();
+  */
```

Comment:
Since we are not using MPI directly, how to call an MPI primitive such as MPI_Wtime() separately in the code to measure the training time?

Reply:
Since this is an illustrative example for future users, let's leave out timing code. For your personal use, you can still call `MPI_Wtime()` before and after whichever function you want to measure the time of right? For example, before and after `pg.allreduce`?

Reply:
I was measuring the time for training. But for the sake of simplicity, I removed it.